src/pages/Task.tsx

```diff
@@ -43,6 +43,7 @@ export const Task: React.FC = () => {
     GetTaskQueryVariables
   >(GET_TASK, {
     variables: { taskId: id, execution: selectedExecution },
+    fetchPolicy: "network-only",
```

Comment:
what does this do?

Reply:
I accidentally branched off of my previous branch and not master, this is from my previous branch. 
It gets all the data from actually performing the query even if there is data cached from a previous query (like from the Tasks page).

Reply:
I fixed the branch issue, it should be easier to review now.